src/options.ts

```diff
@@ -295,7 +295,7 @@ export const getOptions = () => {
     ),
     micromambaVersion: parseOrUndefined(
       'micromamba-version',
-      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+-\d+$/)]),
+      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+(?:\.(?:rc|alpha|beta)\d+)?-\d+$/)]),
```

Comment:
To match PEP440 version numbers of release candidates, alphas, and betas (for instance `2.0.0rc0`):
```suggestion
      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+(?:\.?(?:rc|alpha|beta)\d+)?-\d+$/)]),
```

Reply:
Let's go. :)
```suggestion
      z.union([z.literal('latest'), z.string().regex(/^\d+\.\d+\.\d+(?:\.?(?:rc|alpha|beta)\d*)?-\d+$/)]),
```